CHANGELOG.md

```diff
@@ -4,11 +4,14 @@
 
 ## Important Notes
 
+- [#616](https://github.com/oauth2-proxy/oauth2-proxy/pull/616) Ensure you have configured oauth2-proxy to use the `groups` scope. The user may be logged out initially as they may not currently have the `groups` claim however after going back through login process wil be authenticated.
+
 ## Breaking Changes
 
 ## Changes since v6.1.1
 
 - [#764](https://github.com/oauth2-proxy/oauth2-proxy/pull/764) Document bcrypt encryption for htpasswd (and hide SHA) (@lentzi90)
+- [#616](https://github.com/oauth2-proxy/oauth2-proxy/pull/616) Add support to ensure user belongs in required groups when using the OIDC provider
```

Comment:
Be sure to add your user handle for credit if you want it

Reply:
Ok done and rebased down to one commit, good to go!

![gtg](https://media4.giphy.com/media/ZF3bEnWthZVEYzBeES/giphy.gif?cid=ecf05e479mxzg066jsyqczegf84dxi38xp4x9bvist43jb45&rid=giphy.gif)